lib/config.js

```diff
@@ -10,6 +10,10 @@ export const loadConfig = async (configPath) => {
     )
   }
 
+  if (typeof config.checksum === 'undefined') {
+    config.checksum = 'benthos'
```

Comment:
Can we call this `checksumPrefix`, mainly thinking checksum would be confused with passing an actual checksum.

```suggestion
    config.checksumPrefix = 'benthos_'
```